assets/dev/js/editor/elements/views/base.js

```diff
@@ -112,7 +112,14 @@ BaseElementView = BaseContainer.extend( {
 				break;
 
 			default:
-				ChildView = elementor.modules.elements.views.Widget;
+				const registeredWidgetArgs = elementor.getRegisteredElementType( elType, model.get( 'widgetType' ) );
```

Comment:
Please register the default elements (section, column, container) and remove them from the switch